galaxy.yml

```
@@ -1,7 +1,7 @@
 ---
 namespace: stackhpc
 name: openstack
-version: 0.5.1
+version: 0.5.2
```

```suggestion
version: 0.5.3
```
Now 0.5.2 has been used as a release already: https://github.com/stackhpc/ansible-collection-openstack/releases/tag/0.5.2